phpstan.neon

```diff
@@ -5,6 +5,8 @@ includes:
 parameters:
 	level: 6
 	reportUnmatchedIgnoredErrors: false
+	checkMissingIterableValueType: false
+	checkGenericClassInNonGenericObjectType: false
```

Comment:
This shouldn't happen. I can see you've tried hard to make PHPSTAN work, but I'd rather regenerated the PHPSTAN baseline for this big refactoring PR rather than disabled some checks.

Reply:
I am using the same file found on 5.x branch, I hope that works if it does not I will generate a new file

Reply:
I removed the changes to phpstan, I am working on fixes. 

If I still face issues I will let you know